src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -37,6 +39,36 @@ namespace Microsoft.IdentityModel.Logging
     /// </summary>
     public class LogHelper
     {
+        /// <summary>
+        /// Gets or sets a logger to which logs will be written to.
+        /// </summary>
+        public static IIdentityLogger Logger { get; set; } = NullIdentityModelLogger.Instance;
+
+        /// <summary>
+        /// Indicates whether or the log message header (contains library version, date/time, and PII debugging information) has been written.
+        /// </summary>
+        private static bool HeaderWritten { get; set; } = false;
+
+        /// <summary>
+        /// The log message that indicates the current library version.
+        /// </summary>
+        private static string _versionLogMessage = "Library version of Microsoft.IdentityModel: {0}.";
+
+        /// <summary>
+        /// The log message that indicates the date.
+        /// </summary>
+        private static string _dateLogMessage = "Date: {0}.";
```

Comment:
Do we need the date?

Reply:
ILogger will display the date if configured by the app. 
This logic was copied from (EventSource logging logic)[https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev/src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs#:~:text=.InvariantCulture%2C-,_dateLogMessage,-%2C%20DateTime.] and I kept the date in case app doesn't configure a datetime format. 
The date will be displayed only once before the first Wilson message is logged and only when the log level is set to Trace.